src/jukebox/components/controls/common/evdev_listener.py

```diff
@@ -49,14 +49,12 @@ def _filter_by_device_name(all_devices: List[evdev.InputDevice],
 def find_device(device_name: str, exact_name: bool = True, mandatory_keys: Optional[Set[int]] = None) -> str:
     """Find an input device with device_name and mandatory keys.
 
-    Raises
-
-        #. FileNotFoundError, if no device is found.
-        #. AttributeError, if device does not have the mandatory keys
+    :raise FileNotFoundError: if no device is found.
+    :raise AttributeError: if device does not have the mandatory key
 
     If multiple devices match, the first match is returned
 
-    :param device_name: See :func:`_filter_by_device_name`
+    :param device_name: See :func:`_filter_by_device_name
```

Comment:
Is this change correct?

Reply:
Nope.

although the markdown formatter doesn't pick up the rst syntax, at least the formatting should be closed.

Will fix this.